packages/api-docs-builder/ApiBuilders/ComponentApiBuilder.ts

```diff
@@ -563,7 +562,8 @@ const attachPropsTable = (reactApi: ReactApi) => {
           deprecationInfo:
             renderMarkdownInline(deprecation?.groups?.info || '').trim() || undefined,
           signature,
-          ...(Object.keys(additionalPropsInfo).length === 0 ? {} : { additionalPropsInfo }),
+          additionalInfo:
```

Comment:
It was a typo

| Before | After |
| ------- | ---- |
| <img width="728" src="https://github.com/mui/material-ui/assets/13808724/21d804f5-1a8e-4377-88a5-3484ecae1e5e"> | <img width="692" src="https://github.com/mui/material-ui/assets/13808724/96968286-0ad3-413f-9f21-77986791b986"> |
| <img width="778" src="https://github.com/mui/material-ui/assets/13808724/9e0bcc4a-7d6c-4a8d-b6b8-2bd16929a1b3"> | <img width="751" src="https://github.com/mui/material-ui/assets/13808724/723a5131-f1ce-40f5-8103-f51b57095842"> |
| <img width="450" src="https://github.com/mui/material-ui/assets/13808724/fc8d7b38-1451-43fc-a212-122eb47eebda"> | <img width="808" src="https://github.com/mui/material-ui/assets/13808724/1766bda0-bba4-4d82-aa94-eab3d885ca03"> |
| <img width="767" src="https://github.com/mui/material-ui/assets/13808724/b9535467-a6fb-423c-9566-ceb63dce753d"> | <img width="779" src="https://github.com/mui/material-ui/assets/13808724/12421793-3f29-469e-b442-6f5b81c8445a"> |

Reply:
Similarly here, it'd be great to have these in an `info` callout. Also, I think I'd probably put it _below_ the "Default" text ⎯ it's complementary & generic info whereas "Type" & "Default" are the prop's main pieces of content.

Reply:
About the text itself, how about:
> _See the Slots API section below for more details._

Then, similar to the above, having "the Slots API section" as the link!